### DISCUSSION TITLE
How to integrate Jira?

### DISCUSSION THREAD

**Discussion by gruselglatz** [DependencyTrack/dependency-track#2669] [Q&A]
Idk how to configure the Jira notification. The docs dont give any information either :(

![grafik](https://user-images.githubusercontent.com/15243394/231724881-86a18924-5635-4d11-a8b1-aac8492f4638.png)

**Comment by zts3y** [comment]
Destination = Your JIRA Project Key
Jira Ticket Type = The Issue Type to create (Story, Task, Bug, etc)

**Reply by gruselglatz** [reply]
And where do I configure my jira server? The above fields are mentioned in the docs as well.

**Comment by whiteninja76** [comment]
So for JIRA cloud would the syntax of this be 
destination = https://[companyname].atlassian.net/jira/software/projects/[projectKey] ???

And do i set the PAT token for jira access on <dependecytrackurl>/admin/integrations/jira  
![image](https://github.com/DependencyTrack/dependency-track/assets/34644528/c098d7f6-11bb-40a4-882b-505038700d3e)
